pydatastructs/miscellaneous_data_structures/queue.py

```diff
@@ -0,0 +1,63 @@
+
+__all__ = [
+    'Queue'
+]
+
+class Queue(object):
+    """Respresentation of queue data structure
```

Comment:
```python
"""
Representation of queue data structure.
```

Reply:
[This line](https://github.com/codezonediitj/pydatastructs/blob/master/pydatastructs/miscellaneous_data_structures/stack.py#L12) also needs a change. Can you please do the same in this PR?